### ISSUE TITLE
Is row-level security policy for table implemented ?

### ISSUE THREAD

**Issue by esciullo** [electric-sql/pglite#138]
Is row-level security policy for table implemented ?

After running this script:
```sql
CREATE SCHEMA "playground";

CREATE TABLE IF NOT EXISTS "playground"."app_configurations" (
	"name" varchar PRIMARY KEY NOT NULL,
	"value" text NOT NULL,
	"created_at" timestamp with time zone DEFAULT now(),
	"updated_at" timestamp with time zone,
	"created_by" varchar NOT NULL,
	"updated_by" varchar,
	"is_enabled" boolean DEFAULT true,
	"security_level" integer DEFAULT 0
);

ALTER TABLE playground.app_configurations ENABLE ROW LEVEL SECURITY;

CREATE OR REPLACE FUNCTION playground.get_current_security_level() RETURNS INTEGER LANGUAGE PLPGSQL AS 
$$ 
BEGIN 
  RAISE EXCEPTION 'Not implemented yet!';
END;
$$;

CREATE POLICY app_configurations_security_level ON playground.app_configurations FOR ALL
USING (security_level <= playground.get_current_security_level()) 
WITH CHECK (security_level <= playground.get_current_security_level());
```
I expected that an exception was throwed by the `get_current_security_level()` when quering the `app_configurations` table, but I'm able to query the table.

**Comment by samwillis** [comment]
Hey @esciullo 

Which version of PGlite are you using?

**Comment by esciullo** [comment]
@samwillis `"@electric-sql/pglite": "0.1.5"`

**Comment by samwillis** [comment]
Hey @esciullo

We have just released a new alpha (v0.2.0-alpha.7) that lets you specify the username when starting PGlite.

So you can create a database like normal, create a user, then shut down and restart as that new user - RLS should be fully applied in the context of that user.

```
const db = new PGlite("./pgdata-test-user");
await db.exec(
  "CREATE USER test_user WITH PASSWORD 'bla';"
);
await db.close();

const db2 = new PGlite({
  dataDir: "./pgdata-test-user",
  username: "test_user",
});
// db2 is now connected as test_user
```

**Comment by esciullo** [comment]
Thanks @samwillis, but the issue is still present.
